packages/discord.js/typings/index.test-d.ts

```diff
@@ -361,12 +361,12 @@ client.on('messageCreate', async message => {
   expectAssignable<InteractionCollector<ButtonInteraction>>(buttonCollector);
 
   // Verify that select menus interaction are inferred.
-  const selectMenuCollector = message.createMessageComponentCollector({ componentType: ComponentType.SelectMenu });
+  const selectMenuCollector = message.createMessageComponentCollector({ componentType: ComponentType.StringSelect });
   expectAssignable<Promise<SelectMenuInteraction>>(
```

Comment:
```suggestion
  expectAssignable<Promise<StringSelectMenuInteraction>>(
```
Shouldn't it be `StringSelectMenuInteraction`?

Reply:
it should 
> tests havent been changed since split by classes